src/app/pages/courses-page/course-item/course-item.component.html

```diff
@@ -1 +1,28 @@
-<p>course-item works!</p>
+<section class="course-item-container">
```

Comment:
you can wrap with `ng-container` and use `*ngIf="course"` to prevent rendering if for some reason `course` was not provided

Reply:
Will do, good point